synadm/cli/room.py

```diff
@@ -40,6 +40,16 @@ def join(helper, room_id_or_alias, user_id):
     helper.output(out)
 
 
+@room.command()
+@click.argument("room_alias", type=str)
+@click.pass_obj
+def resolve(helper, room_alias):
```

Comment:
I do like the name actually, but am thinking next steps/future commands: You have a suggestion on how the opposite of such a command could be called? As in getting the room alias from a room id. As this was suggested already, well not directly but as an option to room commands directly showing the alias instead of the id: https://github.com/JOJ0/synadm/issues/25. And thus I think a single command translating id to alias would also be nice and a good first step :-)

Reply:
I agree that resolve would fit both directions. Unsure if it's *really* perfect to use it that way though.
`room_alias` to `room_id` will always only return a single `room_id`, while the opposite direction can yield multiple `room_alias`es for a `room_id`.

Reply:
further brainstorming this morning here ;-) or maybe it could be simply:
```
synadm room getid 'alias'
synadm room getalias 'id'
```
or even
```
synadm room id 'alias'
synadm room alias 'id'
```